server/covmanager/templates/collections/report.html

```diff
@@ -138,24 +210,67 @@
       this.reverse = (this.sortKey === sortKey) ? !this.reverse : false
       this.sortKey = sortKey
     },
+    browse: function(id) {
+      window.open(`../collections/${id}/browse/`, '_blank').focus()
+    },
     summary: function(id) {
       window.open(`../collections/${id}/summary/`, '_blank').focus()
     },
-    collection_click_handler: function(id) {
+    report_click_handler: function(id) {
       var self = this
-      let idx = self.selected_collections.indexOf(id)
+      let idx = self.selected_reports.indexOf(id)
       let target = $("#" + id)
 
       if (idx < 0) {
-        self.selected_collections.push(id)
-        if (target)
-          target.toggleClass("collection-selected", true)
+        self.selected_reports.push(id)
       } else {
-        self.selected_collections.splice(idx, 1)
-        if (target)
-          target.toggleClass("collection-selected", false)
+        self.selected_reports.splice(idx, 1)
       }
     },
+    publish: function (report) {
```

Comment:
As discussed in IRC, `publish()` (and `promote()`) could have confirmation prompts before taking effect. Not a must, but it would make it a little safer from unnoticed, unintended clicks.